config/initializers/okcomputer.rb

```diff
@@ -6,12 +6,37 @@
 # Required
 OkComputer::Registry.register "ruby_version", OkComputer::RubyVersionCheck.new
 OkComputer::Registry.register "redis", OkComputer::RedisCheck.new(url: ENV.fetch("REDIS_URL", Settings.redis_url))
-if Settings.rabbitmq.enabled
-  OkComputer::Registry.register "rabbit",
-    OkComputer::RabbitmqCheck.new(hostname: Settings.rabbitmq.hostname,
+
+class RabbitQueueExistsCheck < OkComputer::Check
+  attr_accessor :queue_names, :conn
```

Comment:
Not a big deal: would an `attr_reader` suffice?

Reply:
I was just cribbing some code, but maybe I cribbed too much.